lib/mqtt/model/status_message.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:noheva_visitor_ui/mqtt/model/abstract_message.dart';
```

Comment:
Replace single quotes with double quotes. It's a shame you apparentally can't configure Dart autoformatter to do it for you.

Same applies for the whole project.